views/win.html.erb

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" class="mdl-js">
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<link href="assets/favicon.ico" rel="icon">
+<link href="assets/css/bootstrap.min.css" rel="stylesheet">
+<link href="assets/css/styles.css" rel="stylesheet">
+<script src="assets/js/jquery.min.js"></script>
+<script src="assets/js/bootstrap.min.js"></script>
+<title>Codebreaker Web</title>
+
+<body>
+  <section id="introduction">
+    <div class="container">
+      <div class="row fill-viewport align-items-center justify-content-center">
+        <div class="col-12 col-md-6">
+          <div class="row">
+            <div class="col-md-auto">
+              <h1 class="text-white title inline-block">Codebreaker 2018</h1>
+            </div>
+          </div>
+          <%= render_partial('./views/shared/_info-panel.html.erb') %>
+          <hr class="mb-4">
+          <div class="row">
+            <div class="text-center">
+              <div class="text-center">
+                <h3 class="text-white">
+                  Congratulations, NAME! You won the game!
```

Comment:
since this is just a suggestion i would skip it for this task

Reply:
But it's very important to use I18n in such cases. Because if you decide to translate your app it will be hard without it. I suggest using it for your views

Reply:
you know, i will not decide :) But gem is added